HomeWork_12/HW12.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime
+def timer():
+    start = datetime()
+    def inner():
+        return datetime() - start
+    return inner
+
+def funcname(func):
+    def inner(*args,**kwargs):
+        print(f"Функція {func.__name__}")
+        return func(*args,**kwargs)
+
+import json
+
+
+
+phonebook = {
+        "Aleks": "347638756",
+        "Fedor": "3784638754",
+        "Den": "2387463745"
+
+    }
+json_phonebook = json.dumps(phonebook)
```

Comment:
Your program should initialize the data from a file. Remove this hardcoded part and add a part where you read the data from a file and transform the data to a dict (json.loads) and write this dict to the `json_phonebook` variable